docs/ai/tutorials/evaluate-safety.md

```diff
@@ -0,0 +1,148 @@
+
+   :::code language="csharp" source="./snippets/evaluate-safety/MyTests.cs" id="GetResponse":::
+
+   The test in this tutorial evaluates the LLM's response to an astronomy question. Since the <xref:Microsoft.Extensions.AI.Evaluation.Reporting.ReportingConfiguration> has response caching enabled, and since the supplied <xref:Microsoft.Extensions.AI.IChatClient> is always fetched from the <xref:Microsoft.Extensions.AI.Evaluation.Reporting.ScenarioRun> created using this reporting configuration, the LLM response for the test is cached and reused.
+
+1. Add a method to validate the response.
+
+   :::code language="csharp" source="./snippets/evaluate-safety/MyTests.cs" id="Validate":::
+
+   > [!TIP]
+   > One of the checks the validation performs is that there are no diagnostics with a severity of warning or higher in each metric. Some of the evaluators, for example, <xref:Microsoft.Extensions.AI.Evaluation.Safety.ViolenceEvaluator>, produce a warning if you only evaluate the response and not the message. Similarly, they expect an equal number of messages and responses in the data you pass to <xref:Microsoft.Extensions.AI.Evaluation.Reporting.ScenarioRunExtensions.EvaluateAsync*>. However, even though an evaluator might produce a warning diagnostic in these cases, it still proceeds with the evaluation.
+
+1. Finally, add the [test method](xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute) itself.
+
+   :::code language="csharp" source="./snippets/evaluate-safety/MyTests.cs" id="TestMethod":::
+
+   This test method:
+
+   - Creates the <xref:Microsoft.Extensions.AI.Evaluation.Reporting.ScenarioRun>. The use of `await using` ensures that the `ScenarioRun` is correctly disposed and that the results of this evaluation are correctly persisted to the result store.
+   - Gets the LLM's response to a specific astronomy question. The same <xref:Microsoft.Extensions.AI.IChatClient> that will be used for evaluation is passed to the `GetAstronomyConversationAsync` method in order to get *response caching* for the primary LLM response being evaluated. (In addition, this enables response caching for the LLM turns that the evaluators use to perform their evaluations internally.)
+   - Runs the evaluators against the response. Like the LLM response, on subsequent runs, the evaluation is fetched from the (disk-based) response cache that was configured in `s_safetyReportingConfig`.
+   - Runs some safety validation on the evaluation result.
+
+## Run the test/evaluation
+
+Run the test using your preferred test workflow, for example, by using the CLI command `dotnet test` or through [Test Explorer](/visualstudio/test/run-unit-tests-with-test-explorer).
+
+## Generate a report
+
+You might choose to generate a report to view the evaluation results. For information about how to do so, see [Generate a report](evaluate-with-reporting.md#generate-a-report).
```

Comment:
It would be better to recommend generating the report since the reporting is a core part of the functionality (and since we are using the Reporting package in this tutorial unlike the quick start example). Please also update the other tutorial if needed...

i.e. 
```suggestion
To generate a report to view the evaluation results see [Generate a report](evaluate-with-reporting.md#generate-a-report).
```